skimage/feature/_hog.py

```diff
@@ -141,9 +152,7 @@ def hog(image, orientations=9, pixels_per_cell=(8, 8), cells_per_block=(3, 3),
     """
     image = np.atleast_2d(image)
 
-    if multichannel is None:
-        multichannel = (image.ndim == 3)
-
+    multichannel = channel_axis is not None
```

Comment:
Don't we need some moveaxis logic here @grlee77 @alexdesiqueira?

Reply:
never mind, just saw that `@channel_as_last_axis` takes care of that for us. Got confused because other functions don't do that. 🙏